Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM github/super-linter
 COPY rules/ /rules/
 COPY entrypoint.sh /entrypoint.sh
+
+# We only want one Dockerfile linter
+ENV VALIDATE_DOCKERFILE=false
+ENV VALIDATE_DOCKERFILE_HADOLINT=true
```

Comment:
This doiesnt quite work, sorry. Form the docs:
```
Note: All the VALIDATE_[LANGUAGE] variables behave in a very specific way:

If none of them are passed, then they all default to true.
If any one of the variables are set to true, we default to leaving any unset variable to false (only validate those languages).
```

Just set VALIDATE_DOCKERFILE to false

Reply:
Oh weird. Thought it's nice to be explicitly, but that's ... interesting behaviour 😄

Reply:
yeah, super weird- surely a default should be a default, not a default unless some other thing is set. Anyway